src/DynamoPackages/Package.cs

```diff
@@ -98,6 +103,15 @@ public bool TypesVisibleInManager
         /// </summary>
         public IEnumerable<string> HostDependencies { get { return hostDependencies; } set { hostDependencies = value; RaisePropertyChanged("HostDependencies"); } }
 
+        private IEnumerable<string> nodeAnnotationPaths;
+
+        public IEnumerable<string> NodeAnnotationPaths
+        {
+            get { return nodeAnnotationPaths; }
+            set { nodeAnnotationPaths = value; RaisePropertyChanged(nameof(NodeAnnotationPaths)); }
+        }
```

Comment:
This was to enable the user to add paths through the pkg.json. This is not being used right now, instead we are only looking for a `doc` folder in the package directory.